kodilla-testing/src/main/java/com/kodilla/testing/collection/OddNumbersExterminator.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.testing.collection;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class OddNumbersExterminator {
+
+    List<Integer> resultNumbers = new ArrayList<>();
+    int exterminatedNumbers = 0;
```

Comment:
to takze powinna byc mienna lokalna. Jesli chcesz zwrocic obie wartosci z metody (List<Integers> oraz int), musisz stworzyc klase opakowujaca ktora bedzie miala 2 pola (resultNumbers oraz exterminatedNumbers) i zwracac instancje tej klasy.
* w razie czego przypomij mi na spotkaniu, to pokaze Ci to